patches/chrome-browser-ui-extensions-extension_action_view_controller.cc.patch

```diff
@@ -1,12 +1,12 @@
 diff --git a/chrome/browser/ui/extensions/extension_action_view_controller.cc b/chrome/browser/ui/extensions/extension_action_view_controller.cc
-index 905f9f30b85458f0b30fb1e871531a2211db181a..e0cf4351310ffd2d4b196b870af669cd071001a4 100644
+index 2d664ea22c78801987449b881f470577af112edb..7ee414d2f5ca3516885c7e4e4479cfdb1623ae6f 100644
 --- a/chrome/browser/ui/extensions/extension_action_view_controller.cc
 +++ b/chrome/browser/ui/extensions/extension_action_view_controller.cc
-@@ -61,7 +61,6 @@ ExtensionActionViewController::ExtensionActionViewController(
+@@ -62,7 +62,6 @@ ExtensionActionViewController::ExtensionActionViewController(
            extensions::ExtensionRegistry::Get(browser_->profile())),
        popup_host_observer_(this),
        weak_factory_(this) {
--  DCHECK(main_bar);
+-  DCHECK(extensions_container);
```

Comment:
why is the DCHECK removed

Reply:
This was introduced here: https://github.com/brave/brave-core/pull/2401/commits/e3ee22926e2280f30c4779524390679503c1a275 with c75 bump.